chat-widget/src/components/livechatwidget/interfaces/ILiveChatWidgetComponentOverrides.ts

```diff
@@ -10,6 +10,7 @@ export interface ILiveChatWidgetComponentOverrides {
     outOfOfficeHoursPane?: ReactNode | string;
     postChatLoadingPane?: ReactNode | string;
     postChatSurveyPane?: ReactNode | string;
+    preChatSurveyPane?: ReactNode | string;
```

Comment:
Why add this now?

Reply:
This is to add support for one of our customers that might need the custom Override for PreChat as well.